### ISSUE TITLE
Option to move unmatched/extraneous files to a different folder

### ISSUE THREAD

**Issue by coreyemtp** [j68k/verifydump#13]
I'm not proposing making this more of a ROM manager, but this ability/option would obviate one of the primary needs I still have with the tool over clrmamepro. Until/if Roman adds RVZ support to clrmamepro, this would be a huge help from an already invaluable tool.

**Comment by j68k** [comment]
I sorta like that the tool never touches your files so there's (hopefully) no way for a bug or user error to make a mess. Just to understand the use case a bit better, would that be for moving old obsolete dumps out of the way, or something else?

**Comment by coreyemtp** [comment]
Correct, those and crap files (.txt, etc.) that might be found in the directory but not matched in the DAT.
